examples/07_shared_task_data.py

```diff
@@ -100,13 +100,14 @@
             # Here we don't use dict initialization.
             td = rp.TaskDescription()
             td.executable     = '/bin/echo'
-            td.arguments      = ['-c', 'input.dat', '%d' % i]
+            td.arguments      = ['$(cat input.dat)', '%d' % i]
```

Comment:
This will not work with `mpirun` and the like.  Should we maybe just run `/bin/cat input.dat`?

Reply:
ok, will do